content/en/docs/armory-enterprise-matrix.md

```diff
@@ -28,7 +28,7 @@ This page describes the features and capabilities that Armory supports. Note tha
 
 ![OSS](/images/oss.svg) The feature or parts of it are available in Open Source Spinnaker.
```

Comment:
open source Spinnaker vs Open Source Spinnaker ??

We seem to use "Open Source" and "open source" interchangeably. Google style guide lists it as "open source" in the [word list](https://developers.google.com/style/word-list#letter-o).